packages/web/src/utils/urlUtils.js

```diff
@@ -1,14 +1,13 @@
 const url = require('url')
 
 const fetchUriParams = (uri, paramName, defaultValue = undefined) => {
-  const params = url.parse(uri, true)
```

Comment:
This doesn't seem like a change based on lint. It's probablly better to put small fixes to logic into another PR, along with a few simple test cases.

Reply:
Lint complains about `url.parse` being deprecated.

Reply:
And it should be functionally equivalent.

Reply:
Can we test it? Or maybe I should say.. did you test it? :)

Reply:
I ran some quick tests in console but didn't write CI test or do anything extensive.

If you're not conferable with the change I can revert it.

Reply:
@kaosat-dev thoughts? i'm not a web developer by trade. ;)

Reply:
@RedHatter please revert. We can address later.